static/js/github-create-branch.js

```diff
@@ -2,8 +2,21 @@ let queriedRepos = [];
 let totalRepos = [];
 let uuid;
 
+//todo add comments here for this strange code
+if(window.location.search.indexOf('close') > 0) {
```

Comment:
Yes, will need some comments on this one. I'm not sure where the `close` is coming from and why are doing this twice?

Reply:
Good news, this is trash now